content/en/workflows/access.md

```diff
@@ -73,35 +72,9 @@ Because workflow [actions][5] connect with external software systems, you may ne
 
 Workflow actions can be authenticated in two ways:
 - Credentials and permissions configured in the integration tile
-- Custom connection credentials
-
-### Integration tile credentials
-
-Credentials and account authentication that you set up in the following Datadog integration tiles automatically propagate to the corresponding actions in Workflows:
-- Jira
-- PagerDuty
-- Slack
-
-Configure the integration tiles by following instructions in [Datadog Integrations][6].
-
-### Connection credentials
+- Connection credentials
 
-Use a custom [connection][7] to authenticate a [generic action][8] or any action for which the integration tile does not offer authentication. The workflow connection object allows you to provide credentials to connect to an integration. The parameters vary by integration.
-
-To connect to an arbitrary service, use the HTTP Connection type and choose from two authentication options:
-- Token-based authentication
-- A username and password combination
-
-### Add credentials to an action
-
-Whether you configured credentials in the integration tile or created a custom connection, Workflows makes your credentials available for each workflow action. To add credentials to an action, follow the steps below:
```

Comment:
I removed this section because I don't think is true now. Actions that use integration tile for credentials don't have a **Connection** drop-down section (I think the **Authentication** drop-down was renamed **Connection**